nodebuilder/p2p/module.go

```diff
@@ -6,8 +6,8 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/node"
 )
 
-// Module collects all the components and services related to p2p.
-func Module(tp node.Type, cfg *Config) fx.Option {
+// ConstructModule collects all the components and services related to p2p.
+func ConstructModule(tp node.Type, cfg *Config) fx.Option {
```

Comment:
The previous Module thing was fine and according to the #958. I understand that the reasoning to change this is because API is now Module. IMO, we should still name those as API. WDYT?